gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/offchainConfig/begin.ts

```diff
@@ -19,31 +19,65 @@ export default class BeginOffchainConfig extends SolanaCommand {
     this.require(!!this.flags.state, 'Please provide flags with "state"')
   }
 
-  execute = async () => {
+  makeRawTransaction = async (signer: PublicKey) => {
     const ocr2 = getContract(CONTRACT_LIST.OCR_2, '')
     const address = ocr2.programId.toString()
     const program = this.loadProgram(ocr2.idl, address)
 
     const state = new PublicKey(this.flags.state)
-    const owner = this.wallet.payer
     const version = new BN(2)
 
     await prompt(`Begin setting Offchain config version ${version.toString()}?`)
```

Comment:
`makeRawTransaction` does not send any tx. This should be moved into the `execute`